packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4934,16 +4965,69 @@ class Row extends Flex {
   /// must not be null.
   const Row({
     super.key,
-    super.mainAxisAlignment,
-    super.mainAxisSize,
-    super.crossAxisAlignment,
-    super.textDirection,
-    super.verticalDirection,
-    super.textBaseline, // NO DEFAULT: we don't know what the text's baseline should be
-    super.children,
-  }) : super(
-    direction: Axis.horizontal,
-  );
+    this.mainAxisAlignment = MainAxisAlignment.start,
+    this.mainAxisSize = MainAxisSize.max,
+    this.crossAxisAlignment = CrossAxisAlignment.center,
+    this.textDirection,
+    this.verticalDirection = VerticalDirection.down,
+    this.textBaseline,
+    this.clipBehavior = Clip.none,
+    this.children = const <Widget>[],
+  });
+
+  @override
+  Axis get direction => Axis.horizontal;
+
+  @override
+  final MainAxisAlignment mainAxisAlignment;
+
+  @override
+  final MainAxisSize mainAxisSize;
+
+  @override
+  final CrossAxisAlignment crossAxisAlignment;
+
+  @override
+  final TextDirection? textDirection;
+
+  @override
+  final VerticalDirection verticalDirection;
+
+  @override
+  final TextBaseline? textBaseline;
+
+  @override
+  final Clip clipBehavior;
+
+  @override
+  final List<Widget> children;
+
+  @override
+  Widget build(BuildContext context) {
+    Widget result = Flex(
+      direction: direction,
+      mainAxisAlignment: mainAxisAlignment,
+      mainAxisSize: mainAxisSize,
+      crossAxisAlignment: crossAxisAlignment,
+      textDirection: textDirection,
+      verticalDirection: verticalDirection,
+      textBaseline: textBaseline,
+      clipBehavior: clipBehavior,
+      children: children,
+    );
+
+    // Selection is only enabled when there is a parent registrar.
+    final SelectionRegistrar? registrar = SelectionContainer.maybeOf(context);
+    if (registrar != null) {
+      result = SelectionContainer(
+        registrar: registrar,
+        delegate: StaticMultiSelectableSelectionContainerDelegate(selectedContentSeparator: ' '),
```

Comment:
I think StaticMultiSelectableSelectionContainerDelegate will need to be kept as state. otherwise, the delegate will gets recreated and the state inside it would be needed to be transfer again and again. Can you create another wrapper statefulwidget that hold the StaticMultiSelectableSelectionContainerDelegate as its state?